chapters/en/slides/module5/module5_05.Rmd

```diff
@@ -0,0 +1,393 @@
+
+In addition to losing the overall intuitiveness of the color scheme,
+the many hues mean that we’re likely
+to perceive changes in the color in a non-uniform manner,
+and see groups where they are none.
+
+Importantly,
+the rainbow map we have used here is a modern color scheme called “turbo”,
+which was recently developed by Google.
+Most rainbow color schemes you will see on plots in the wild use a color scheme called “jet”,
+which used to be the default in many plotting packages
+but is not a good choice for any data,
+because it introduces artificial banding and highlighting of values.
+
+The take home message here is that there are valid use cases for modern rainbow maps,
+but you should be confident that there is no better alternative before reaching for this tool,
+and be careful when interpreting rainbow colored plots in general.
+
+
+The jet rainbow colormap should be avoided for many reasons,
+including that the sharp transitions between colors introduces visual threshold
+that do not represent the underlying continuous data.
+Another issue is luminance (brightness).
+For example,
+your eye is drawn to the yellow and cyan regions,
+because the luminance is higher.
+This can have the unfortunate effect of highlighting features in your data
+that don't exist,
+misleading your viewers!
+Since higher values are not always lighter,
+this means that your graph is not going to translate well to greyscale.
+More details about jet can be found in [this blog post] and [this series of posts].
+A better alternative when you really need small differences in your data to stand out
+is to use the [turbo rainbow color scheme](https://ai.googleblog.com/2019/08/turbo-improved-rainbow-colormap-for.html).
+
+[this blog post]: https://jakevdp.github.io/blog/2014/10/16/how-bad-is-your-colormap/
+[this series of posts]: https://mycarta.wordpress.com/2012/05/12/the-rainbow-is-dead-long-live-the-rainbow-part-1/
+
+
+---
+
+## Diverging color schemes are suitable for plots with a natural midpoint.
+
+<center><img src=/module5/color-schemes-lightness.png width=70%></img></center>
+
+Notes:
+In this slide,
+you see the lightness variation among all the color schemes we have covered.
+Categorical colors are discretely separated,
+perceptually uniform color schemes vary lightness linearly throughout the range,
+whereas rainbow maps
+peak lightness in the middle and have dark colors at the extremes.
+
+The color scheme to the far right is called a diverging color scheme,
+which are useful when there is a natural midpoint in the data
+as we will see next.
+
+There is also one other type of color scheme,
+cyclical,
+which starts and ends in both the same lightness and hue.
+This design is useful for values with natural cycles
+such as degrees of an angle of month of the year,
+where the end and beginning are close to each other.
+
+TODO technically this is the wrong blue, the vega one is completely stright I belive
```

Comment:
Reminder comment to address this.

Reply:
This is minor so leaving as is even if it mightt be slightly wrong technically.